BUILD.md

```diff
@@ -80,33 +80,30 @@ The dependencies can be installed through the macOS package manager [Homebrew](h
     # substitute the paths to your OpenSSL 1.1 library
     export SSL_LIB_PATH=/usr/local/Cellar/openssl@1.1/1.1.1l/lib/
     export CPATH=/usr/local/Cellar/openssl@1.1/1.1.1l/include/
-    python setup.py build --force
+
+    pip install build
+    python3 -m build
+
+<!-- TODO: update all build cmds -->
```

Comment:
We need to make sure that all build related documentation is changed. Does this TODO still need to be done? For example, it looks like the "build" Python package needs to be added to the dependencies section.

Reply:
Hey Dylan, that TODO is outdated since I already updated all references to the command that builds the client (at least in this repository), so I'll remove it soon in this PR.

And I don't think the build package needs to be listed in the dependencies section because that section describes distro specific dependencies, and the build package isn't specific to any of those distros. I'm planning to move all the build dependencies into a requirements.txt [later on](https://aerospike.atlassian.net/jira/software/c/projects/CLIENT/issues/CLIENT-1880).

Reply:
sounds good